scripts/build/ci-msi-build/oss/resources/license/LICENSE.md

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Do we need the license files?

Reply:
We do - it will be included in the "agree" dialog during the installation of the MSI